beluga_amcl/docs/PARAMETERS.md

```diff
@@ -0,0 +1,75 @@
+| <nobr>`initial_pose.covariance_[x, y, yaw, xy, xyaw, yyaw]`</nobr> | Covariance to use with the initial pose when initializing the particle filter. _Navigation 2 considers these to be zero._ | | ✅ |
+| `save_pose_rate` | Maximum rate (Hz) at which to store the last estimated pose and covariance to the parameter server. _ROS 2 does not have a parameter server, this parameter is mentioned in the documentation but it can't be implemented._ | | |
+| `always_reset_initial_pose` | Requires that AMCL is provided an initial pose either via topic or `initial_pose*` parameter when reset. Otherwise, by default AMCL will use the last known pose to initialize. | ✅ | ✅ |
+| `first_map_only` | When set to false, allows AMCL to accept maps more than once on the `map` topic. | ✅ | ✅ |
+| `tf_broadcast` | Set this to false to prevent AMCL from publishing the transform between the global frame and the odometry frame. | ✅ | ✅ |
+| `transform_tolerance` | Time with which to post-date the transform that is published to indicate that this transform is valid into the future. | ✅ | ✅ |
+
+## Particle Filter Parameters
+
+| Parameter | Description | Navigation 2 AMCL | Beluga AMCL |
+|-----------|-------------|:-----------------:|:-----------:|
+| `max_particles` | Maximum allowed number of particles. | ✅ | ✅ |
+| `min_particles` | Minimum allowed number of particles. | ✅ | ✅ |
+| `pf_err` | Maximum particle filter population error between the true distribution and the estimated distribution. It is used in [KLD resampling](fox2001) to limit the allowed number of particles to the minimum necessary. | ✅ | ✅ |
+| `pf_z` | Upper standard normal quantile for P, where P is the probability that the error in the estimated distribution will be less than `pf_err` in [KLD resampling](fox2001). | ✅ | ✅ |
+| `spatial_resolution_[x, y, theta]` | Spatial resolution to create buckets for [adaptive KLD resampling][fox2001].                                | | ✅ |
+| `recovery_alpha_fast` | Exponential decay rate for the fast average weight filter, used in deciding when to recover from a bad approximation by adding random poses. | ✅ | ✅ |
+| `recovery_alpha_slow` | Exponential decay rate for the slow average weight filter, used in deciding when to recover from a bad approximation by adding random poses. | ✅ | ✅ |
+| `resample_interval` | Number of filter updates required before resampling. | ✅ | ✅ |
+| `selective_resampling` | When set to true, it will reduce the resampling rate when not needed and help avoid particle deprivation. The resampling will only happen if the effective number of particles $(N_{eff} = 1/ {\sum k_i^2})$ is lower than half the current number of particles. | ✅ | ✅ |
+| `update_min_a` | Rotational movement required from last resample for resampling to happen again. _Navigation 2 skips the filter update completely while Beluga only skips the resampling step. Sensor and motion updates are not skipped._ | ✅ | ✅ |
```

Comment:
Not necessarily for this PR, but I think we should standarize both the names we use to refer to ourselves (Beluga library, Beluga node, `beluga_amcl`) and Nav2's (Navigation 2, Nav 2 AMCL, `nav2_amcl`).

In this particular case, "Navigation 2" is too broad, since that denomination goes much further than just `nav2_amcl`.

Reply:
Replaced all relevant instances with "Nav2 AMCL2 in https://github.com/Ekumen-OS/beluga/pull/196/commits/b0672a7827aa45034ea11d6fa2404de3df918006.